package.json

```diff
@@ -75,9 +88,9 @@
     "highlight.js": "^8.6.0",
     "marked": "^0.3.3",
     "mocha": "^2.2.5",
-    "page": "^1.6.3",
+    "page": "1.6.3",
```

Comment:
`page` is pinned because of a navigation issue in the latest version where you cannot navigate on the sidebar.